export/main.ts

```diff
@@ -99,19 +108,27 @@ const input = [];
 for(const proj of projects) {
     input.push(limit(async () => {
         const data = await fetchProjectData(proj.id, proj.group_name)
+        let msg = "";
+
+        if(data) {
+            msg = data.name ? data.name.padEnd(50, ' ') : "";
```

Comment:
```suggestion
            msg = data.name ? data.name.padEnd(50, ' ') : "".padEnd(50, ' ');
```
Just a suggestion...might be a matter of opinion/style...but this would keep the output from changing widths (aka keep the completion metrics at the same spot on the screen).